.do/values.yaml

```diff
@@ -94,6 +94,21 @@ codacy-ingress:
       api: <codacy-api.dns.internal>
     spa:
       serviceName: codacy-spa # this will be moved to the ingress default configs when the spa is released
+    annotations:
+      kubernetes.io/ingress.class: nginx
+      kubernetes.io/tls-acme: "true"
+      cert-manager.io/acme-challenge-type: http01
+      cert-manager.io/cluster-issuer: letsencrypt-codacy
+      external-dns.alpha.kubernetes.io/hostname: '<codacy-app.dns.internal>,<codacy-api.dns.internal>'
+      external-dns.alpha.kubernetes.io/scope: public
+      nginx.ingress.kubernetes.io/proxy-body-size: 128m
+      nginx.ingress.kubernetes.io/ssl-redirect: "true"
+      nginx.ingress.kubernetes.io/use-regex: "true"
+    tls:
+      hosts:
+        - host: <codacy-app.dns.internal>
+        - host: <codacy-api.dns.internal>
+      secretName: codacy-ingress-tls-secret
```

Comment:
This enables https on our codacy-ingress with letsencrypt provisioned certificates through cert-manager.